main.js

```diff
@@ -0,0 +1,113 @@
+const API_Key = 'L2uYtoQQwJS67ja8rwrJF9gfl1Bi8vZk';
+
+const mainContainer = document.querySelector('.container-select-city');
+const city_field = document.querySelector('#city-field');
+const getWeatherBtn = document.querySelector('.get-weather');
+const childDiv = document.createElement('div');
+mainContainer.append(childDiv);
+
+const xhr = new XMLHttpRequest();
+
+const cityData = {
+    Lviv: 324561,
+    Kyiv: 324505,
+    Kharkiv: 323903,
+    Chernihiv: 322162,
+    Odesa: 325343,
+    Mariupol: 323037,
+    Poltava: 325523,
+    Zhytomyr: 326609,
+    Cherkasy: 321985,
+    London: 328328,
+    Warsaw: 274663,
+    "New York": 349727,
+};
+
+let counter = 0;
+
+function validateIfCity() {
+    for (let city in cityData) {
+        if (city_field.value === city) {
+            return cityData[city];
+        }
+    }
+    return;
+}
+
+function convertToCelsius(temp) {
+    return ((temp - 32) * 5) / 9;
+}
+
+function loadData(id) {
+    xhr.open(
+        "GET",
+        `http://dataservice.accuweather.com/forecasts/v1/daily/5day/${id}?apikey=${API_Key}`
+    );
+    xhr.send();
+    xhr.onload = function () {
+        if (xhr.status !== 200) {
+            alert (`Error ${xhr.status}: ${xhr.statusText}`);
+        } else {
+            const result = JSON.parse(this.response);
+            showWeather(result);
+        }
+    };
+    xhr.onerror = function () {
+        alert ("Request failed");
+    }
+} 
+
+function showWeather(data) {
+const childDIV = document.createElement("div");
+mainContainer.append(childDIV);
+
+const weather = data.DailyForecasts;
+
+for (let param in weather) {
```

Comment:
Careful with `for...in` loop

If you only want to consider properties attached to the object itself, and not its prototypes, you can use other techniques